.github/workflows/ci.yaml

```diff
@@ -124,20 +135,7 @@ jobs:
           files: |
             mirrord/agent/Dockerfile
             mirrord/cli/Dockerfile
-      # Detect Windows-related changes so Linux tests run when Windows code changes
-      - name: get Windows-related changes
-        id: changed-windows
-        uses: metalbear-co/changed-files@9200e69727eb73eb060652b19946b8a2fdfb654b
-        with:
-          files: |
-            mirrord/layer-win/**
-            mirrord/str-win/**
-            mirrord/**/windows/**
-            mirrord/**/win32/**
-            mirrord/**/win/**
-            scripts/**/*.ps1
-            **/*.ps1
-            .github/workflows/windows-build.yaml
```

Comment:
Let's keep the separate `changed-windows` output, so that we can avoid unnecessarily running non-windows checks, if only windows-specific files changed.